File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        if (nums.size() == 0) return 0;
        int temp = nums[0];
        vector<int> arr;
        arr.push_back(nums[0]);
        for(int i =1 ; i<nums.size() ;i++)
        {
                if(temp != nums[i])
                {
                    arr.push_back(nums[i]);
                    temp = nums[i];


                }
        }
        
        nums=arr;
        return arr.size();
        
    }
};

File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        n = len(nums)
        if n ==0:
            return 0

        k=1

        for i in range (1 ,n):
            if (nums[i]!=nums[k-1]):
                nums[k]=nums[i]
                k+=1

        return k